shell/common/shell_unittests.cc

```diff
@@ -1410,8 +1357,7 @@ TEST_F(ShellTest, WaitForFirstFrameZeroSizeFrame) {
 
   RunEngine(shell.get(), std::move(configuration));
   PumpOneFrame(shell.get(), {1.0, 0.0, 0.0});
-  fml::Status result =
-      shell->WaitForFirstFrame(fml::TimeDelta::FromMilliseconds(1000));
+  fml::Status result = shell->WaitForFirstFrame(fml::TimeDelta::Zero());
```

Comment:
Should this be `fml::TimeDelta::Max()` ?

Reply:
Ah, sorry, I missed the ASSERT below this line saying that the wait is expected to timeout. So, is this testing that `PumpOneFrame` doesn't do anything for a zero size frame, and that's why the wait will time out? Is there another way for `PumpOneFrame` to communicate that it isn't going to do anything?

Reply:
My understanding is that this test wants to assert that `WaitForFirstFrame` will time out if you pump a zero sized frame.

I think a better place to test this would be e.g. in the animator unittests, making sure that `OnAnimatorDidDraw` isn't called under this condition.